learning_paths/api/v1/serializers.py

```diff
@@ -0,0 +1,70 @@
+"""
+Serializer for LearningPath.
+"""
+from rest_framework import serializers
+
+from learning_paths.models import LearningPath
+
+DEFAULT_STATUS = "active"
+IMAGE_WIDTH = 1440
+IMAGE_HEIGHT = 480
+
+
+class LearningPathAsProgramSerializer(serializers.ModelSerializer):
+    """
+    Serialize LearningPath as a Program to be ingested by course-discovery.
+
+    Mocked data example:
+    https://github.com/openedx/course-discovery/blob/d6a57fd69479b3d5f5afb682d2668b58503a6af6/course_discovery/apps/course_metadata/data_loaders/tests/mock_data.py#L580
+    """
+
+    name = serializers.CharField(source="display_name")
+    marketing_slug = serializers.SerializerMethodField()
+    title = serializers.CharField(source="display_name")
+    subtitle = serializers.CharField(source="description")
```

Comment:
Good catch! Even `blank=True, null=False` is not enough because the blank API response raises the error. I added the field as non-blank non-null, same as the title field.